src/aria/widgets/container/Dialog.js

```diff
@@ -609,7 +609,7 @@ module.exports = Aria.classDefinition({
 
             this._calculatePosition();
 
-            if (cfg.modal) {
+            if (cfg.focusIfModal && cfg.modal) {
```

Comment:
Hi,

If case of modal, and if the parameter is false, it would be nice to blur the active field by `Aria.$window.document.activeElement.blur()`. Otherwise, an active element under the mask can potentially stay focused under a modal.